src/Arrays/array_operations.jl

```diff
@@ -42,3 +42,36 @@ function same_sign(A::AbstractArray{N}; optimistic::Bool=false) where {N}
         end
     end
 end
+
+"""
+    _rationalize(::Type{T}, x::AbstractVecOrMat{N}, tol::Real) where {T<:Integer, N<:AbstractFloat}
+
+Approximate an array of floating point numbers as a rational vector with entries
+of the given integer type.
+
+### Input
+
+- `T`   -- (optional, default: `Int`) integer type to represent the rational
+- `x`   -- vector of floating point numbers
+- `tol` -- (optional, default: `eps(x)`) tolerance the result at entry `i`
+           will differ from `x[i]` by no more than `tol`
+
+### Output
+
+An array of type `Rational{T}` where the `i`-th entry is the rationalization
+of the `i`-th component of `x`.
+
+### Notes
+
+See also [`Base.rationalize`](@ref).
+"""
+function _rationalize(::Type{T}, x::AbstractVecOrMat{N}, tol::Real) where {T<:Integer, N<:AbstractFloat}
```

Comment:
The dispatch strategy for `_rationalize` follows that in Julia Base (see [here](https://github.com/JuliaLang/julia/blob/f9720dc2ebd6cd9e3086365f281e62506444ef37/base/rational.jl#L216)); I used the underscore because LazySets doesn't "own" neither the type of the arguments nor `rationalize`. On the other hand, the new function over LazySets is called `rationalize`.